### PR TITLE
Fixes #212: Added java test for spark types

### LINKED ISSUE #212: Add Java Tests
Add Java Tests

### PR DESCRIPTION
**Pull request by utnaf**
Fixes #212 

Added Java tests for Spark Types

### CODE REVIEW

File: src/test/java/org/neo4j/spark/SparkConnectorSuiteIT.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package org.neo4j.spark;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+
+})
```
Comment (conker84) [inline src/test/java/org/neo4j/spark/SparkConnectorSuiteIT.java]: i guess that you missed this?
```suggestion
@Suite.SuiteClasses({DataSourceReaderTypesTSE.class})
```

**Review by conker84 — CHANGES_REQUESTED** [review]
thx @utnaf please look at the comment

File: src/test/java/org/neo4j/spark/SparkConnectorSuiteIT.java (hunk 1 above)
Comment (utnaf) [reply src/test/java/org/neo4j/spark/SparkConnectorSuiteIT.java]: Of course I did 😓

**Comment by conker84** [comment]
@utnaf thank you!

**Review by conker84 — APPROVED** [review]

**Pull request merged**